yearn/contracts/ConvexStrategy.sol

```diff
@@ -0,0 +1,612 @@
+    ///         capture the free return this strategy has generated since the
+    ///         last time its core position(s) were adjusted. Examples include
+    ///         unwrapping extra rewards. This call is only used during normal
+    ///         operation of a strategy, and should be optimized to minimize
+    ///         losses as much as possible. This strategy implements the
+    ///         aforementioned behavior by getting CRV, CVX, and optional extra
+    ///         rewards from the Convex reward pool, using obtained tokens to
+    ///         buy one of the Curve pool's accepted token, and depositing that
+    ///         token back to the Curve pool. This way the strategy is gaining
+    ///         new vault's underlying tokens thus making the profit. A small
+    ///         portion of CRV rewards (defined by keepCRV param) is transferred
+    ///         to the voter contract to increase CRV boost and get more CRV
+    ///         rewards in the future.
+    /// @param debtOutstanding Will be 0 if the strategy is not past the
+    ///        configured debt limit, otherwise its value will be how far past
+    ///        the debt limit the strategy is. The strategy's debt limit is
+    ///        configured in the vault.
+    /// @return profit Amount of realized profit.
+    /// @return loss Amount of realized loss.
+    /// @return debtPayment Amount of repaid debt. The value of debtPayment
+    ///         should be less than or equal to debtOutstanding. It is okay for
+    ///         it to be less than debtOutstanding, as that should only used as
+    ///         a guide for how much is left to pay back. Payments should be
+    ///         made to minimize loss from slippage, debt, withdrawal fees, etc.
+    function prepareReturn(uint256 debtOutstanding)
+        internal
+        override
+        returns (
+            uint256 profit,
+            uint256 loss,
+            uint256 debtPayment
+        )
+    {
+        // Get the initial balance of the vault's underlying token under
+        // strategy management.
+        uint256 initialWantBalance = balanceOfWant();
+
+        // Get CRV and CVX rewards from the Convex reward pool. Also, claim
+        // extra rewards if such rewards are distributed by the pool.
+        IConvexRewardPool(tbtcConvexRewardPool).getReward(address(this), true);
+
+        // Buy WBTC using obtained CRV tokens.
+        uint256 crvBalance = IERC20(crv).balanceOf(address(this));
+        if (crvBalance > 0) {
+            // Deposit a portion of CRV to the voter to gain CRV boost.
+            crvBalance = adjustCRV(crvBalance);
+
+            IERC20(crv).safeIncreaseAllowance(uniswap, crvBalance);
+
+            address[] memory path = new address[](3);
+            path[0] = crv;
+            path[1] = weth;
+            path[2] = wbtc;
+
+            IUniswapV2Router(uniswap).swapExactTokensForTokens(
+                crvBalance,
+                uint256(0),
```

Comment:
uint256(0) -> 0. We have a couple of occurances of this contract.

Reply:
https://github.com/keep-network/tbtc-v2/pull/10/commits/23b2b4753aea07ae8f6e17c0bdebe75974a7d41b